core/src/commitment.rs

```diff
@@ -35,13 +35,22 @@ impl BlockCommitment {
 pub struct BlockCommitmentCache {
     block_commitment: HashMap<Slot, BlockCommitment>,
     total_stake: u64,
+    slot: Slot,
```

Comment:
What if we stored an `Arc<Bank>` instead and allowed direct access to the bank from the commitment cache?

Reply:
Hmm, yeah, should solve the rpc issue. Trying...